clustered/server/service/src/main/java/org/ehcache/clustered/server/state/EhcacheStateServiceProvider.java

```diff
@@ -64,16 +63,16 @@ public boolean initialize(ServiceProviderConfiguration configuration, PlatformCo
     Collection<OffHeapResources> extendedConfiguration = platformConfiguration.getExtendedConfiguration(OffHeapResources.class);
     if (extendedConfiguration.size() > 1) {
       throw new UnsupportedOperationException("There are " + extendedConfiguration.size() + " OffHeapResourcesProvider, this is not supported. " +
-                                              "There must be only one!");
+        "There must be only one!");
     }
     Iterator<OffHeapResources> iterator = extendedConfiguration.iterator();
     if (iterator.hasNext()) {
       offHeapResourcesProvider = iterator.next();
       if (offHeapResourcesProvider.getAllIdentifiers().isEmpty()) {
-        throw new UnsupportedOperationException("There are no offheap-resource defined, this is not supported. There must be at least one!");
+        LOGGER.warn("No offheap-resource defined - this will prevent provider from offering any EhcacheStateService.");
       }
     } else {
-      LOGGER.warn("No offheap-resource defined - this will prevent provider from offering any EhcacheStateService.");
+      throw new UnsupportedOperationException("There are no offheap-resource defined, this is not supported");
```

Comment:
Same bugfix we did in EE: `OffHeapResources` is ALWAYS there, but with 0..N resources inside now.